### PR TITLE
MD3, updated gradle & dependencies

### PR DESCRIPTION
**Pull request by the-weird-aquarian**
Hello again,

This PR includes:

- migrated to material design 3 (no dynamic colors, app will stay with brand colors)
- targeted SDK 32
- updated gradle and dependencies
- fixed some deprecated code and warnings

### CODE REVIEW

File: SimpleLogin/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
             <action android:name="android.intent.action.VIEW" />
             <data android:scheme="mailto" />
 
-            <category android:name="android.intent.category.DEFAULT" />
+            <!--<category android:name="android.intent.category.DEFAULT" />-->
```
Comment (ntnhon) [inline SimpleLogin/app/src/main/AndroidManifest.xml:18]: Can you please explain why removing this intent?

File: SimpleLogin/app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,55 +1,62 @@
...
         <item name="android:textColor">@color/colorDarkGray</item>
     </style>
 
-    <style name="SLMaterialAlertButtonNegative" parent="Widget.MaterialComponents.Button.TextButton">
+    <style name="SLMaterialAlertButtonNegative" parent="Widget.Material3.Button.TextButton.Dialog">
         <item name="android:textColor">@color/colorNegative</item>
     </style>
 
-    <style name="SLMaterialAlertButtonPositive" parent="Widget.MaterialComponents.Button.UnelevatedButton">
+    <style name="SLMaterialAlertButtonPositive" parent="Widget.Material3.Button.TextButton.Dialog.Flush">
```
Comment (ntnhon) [inline SimpleLogin/app/src/main/res/values/styles.xml:44]: This causes the "Yes, sign me out" button invisible in light mode. Any other type that we can use?

**Review by ntnhon — COMMENTED** [review]
Hello again,
Thank your for the PR it looks great. I left some comments.
And also the sidebar menu now renders darker titles as well as dividers that don't reach the borders of the sidebar. Do you know which causes that behavior?

**Comment by the-weird-aquarian** [comment]
> The sidebar menu now renders darker titles as well as dividers that don't reach the borders of the sidebar

Yes that's a design change by Google. Take a look at the images in the [Material3 Navigation Drawer Docs](https://m3.material.io/components/navigation-drawer/overview). Feel free to check out the other Material3 components

File: SimpleLogin/app/src/main/res/values/styles.xml (hunk 2 above)
Comment (the-weird-aquarian) [reply SimpleLogin/app/src/main/res/values/styles.xml]: Oops, my bad. Looks like I missed this somehow. It's fixed now.

File: SimpleLogin/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (the-weird-aquarian) [reply SimpleLogin/app/src/main/AndroidManifest.xml:18]: It didn't allow me to include both intents there. Even with no changes from my side, it gives manifest error. If I'm not wrong this is required when we select "Begin composing with default email" from the dialog after clicking "Send email". In my testing it works as expected. Feel free to check this at your end too. :)

**Comment by ntnhon** [comment]
I tested and found no more issue for far. Thanks a lot for your contribution!
I'd like to ask you something in private but can't find your contact information. Can you please email me at nhon@simplelogin.io?

**Pull request merged**